.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Create new doctype template
+about: For issues where a templateer needs to create a new doctype template.
+title: Create {doctype} template
+labels: templates
+assignees: ''
+
+---
+
+# Create {doctype} template
+
+## Description
+### Doctype Name: {doctype}
+### Directory: https://github.com/thegooddocsproject/templates/{directory_name}
+### Sub-tasks
+1. Create {doctype}-template.md and write content for it
+2. Create {doctype}-guide.md and write content for it
+
+{Optional: add any further explanation if the issue is not self-explanatory}
+
+## Helpful References
+{Optional: Add links to anything that might help the templateer write this particular type of template. For example, a Medium article about how to write good tutorials.}
```

Comment:
**Suggestion (non-blocking):** Perhaps use "Helpful Resources" instead.

Reply:
That sounds good. I'll make that change.